game-core/src/main/java/games/strategy/triplea/attachments/TerritoryAttachment.java

```diff
@@ -474,14 +480,29 @@ private void setWhenCapturedByGoesTo(final ArrayList<String> value) {
     m_whenCapturedByGoesTo = value;
   }
 
-  public ArrayList<String> getWhenCapturedByGoesTo() {
+  private ArrayList<String> getWhenCapturedByGoesTo() {
```

Comment:
With this method now private (and all other usages not using ArrayList explicitly) could we set the type to just use the List (or even Collection) interface?
As long as the implementation isn't changed this shouldn't be an issue until the next incompatible release.

Reply:
Agreed it would be cleaner, but it would require changing the type of the `m_whenCapturedByGoesTo` field.  I believe we did something similar in another PR after we convinced ourselves there would be no compatibility issues.  However, I'd prefer to do that outside of this PR Just In Case. :smile:  (In fact, we should probably do that for all such fields in a single PR.)

I did change the return type of `getCaptureOwnershipChanges()` to `Collection` to reflect that we truly don't need an ordered collection.  5ebb61162